tests/kubic/networking.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test network connectivity
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
```

Comment:
why utils?

Reply:
yeah, we don't need this.  Let me remove this.